src/components/SecondaryPanes/XHRBreakpoints.css

```diff
@@ -82,11 +82,36 @@
 }
 
 .xhr-input-method {
-  display: none;
+  visibility: hidden;
+  margin-inline-start: 4px;
+  padding-block: 0;
+  padding-inline-start: 2px;
+  padding-inline-end: 14px;
+  border: none;
+  border-radius: 2px;
+  font: inherit;
+  color: inherit;
+  -moz-appearance: none;
+  appearance: none;
+  background-color: var(--toolbarbutton-background);
+}
+
+.xhr-input-container.focused::after {
```

Comment:
I'm not fond of putting this graphical element as a pseudo-child of `.xhr-input-container` and with a somewhat arbitrary coordinate (`right: 25px`). This is likely to break at the first change. We should move this `::after` much closer to the `<select>` element, and make sure it's positioned relative to the `<select>` element and not other elements.

Reply:
I made a wrapper span for the select element.